ledger-core/v2/virtual/object/pending_table.go

```diff
@@ -0,0 +1,75 @@
+package object
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/v2/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/v2/reference"
+)
+
+type pendingTable struct {
+	Ordered   pendingList
+	Unordered pendingList
+}
+
+type pendingList struct {
+	oldestPulse pulse.Number
+	requests    map[reference.Global]*struct{
+		active bool
+	}
+}
+
+func newPendingTable() pendingTable {
```

Comment:
Private structures can't be used outside of package. Is it intentional?

Reply:
If not please consider adding a corresponding interface. Unit tests should test this interface.

Reply:
yeah, it was